.github/workflows/testing.yml

```
@@ -1,42 +1,51 @@
-
-    - name: Install dependencies
-      run: uv sync
-
-    - name: Run tests
-      run: pytest -v 
\ No newline at end of file
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)